bots/parsebot.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
```

Comment:
We can order the import statements to be in alphabetical order of some sort (ASC for example) and let the from statements be below the strict imports.

The idea is just code structure as the code base grows, we can have strategies to quickly navigate it.

Reply:
Resolved in latest commit.